student-bios/NazgulM.md

```diff
@@ -0,0 +1,48 @@
+# About me
+
+## Nazgul Maniiasova
+
+<img src="Naza.jpg" width="150" height="100">
```

Comment:
This is a HTML tag and it is not working with Markdown.
To fix it:
`![My photo](assets/Naza.jpg)`